helmfile/upstream/kube-prometheus-stack/values.yaml

```diff
@@ -640,7 +640,7 @@ grafana:
   ## Timezone for the default dashboards
   ## Other options are: browser or a specific timezone, i.e. Europe/Luxembourg
   ##
-  defaultDashboardsTimezone: utc
+  defaultDashboardsTimezone: ""
```

Comment:
not necessary as I see you already modified the `helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl`

Reply:
Why not have it the same everywhere?

Reply:
if we update the upstream chart this file will be overwritten
we use kube-prometheus-stack-sc.yaml.gotmpl to change any of the upstream chart defaults